packages/fx-core/templates/plugins/resource/simpleauth/bicep/input_param.template.bicep

```diff
@@ -0,0 +1,5 @@
+
+param simpleAuth_sku string = 'F1'
```

Comment:
Can you help confirm with Junjie whether we need to keep this parameter? It's strange that only part of resources expose the SKU parameter. The main concerns maybe some project sets the SKU manually, but I'm not sure whether only internal users (us) are doing so.

Reply:
Confirmed with @MuyangAmigo, we can make sku as optional parameters with default value set to "F1".

Reply:
@blackchoey let me discuss with you for a comprehensive format and best practice of parameters.json file.

Reply:
Comfirmed with @MuyangAmigo to keep this skuName as output since it is related to cost which user will be interested. I will sum up a list of all resource plugins' output for review.

Reply:
Can we just remove this parameter? And let user modify the SKU in bicep file if necessary?

Reply:
I will sum up a list of input parameters / output parameters for all resource plugins for review. Let's leave the discussion to that.

Reply:
ok